content/en/docs/developer_guide/flows/flow_triggers/oas_flow_trigger.md

```diff
@@ -5,8 +5,6 @@ weight: 10
 date: 2021-10-27
 ---
```

Comment:
The whole file needs more work. We have to discuss what we want to include here, as Jamie already did include a portion of what we would normally find here under https://deploy-preview-69--api-builder-docs.netlify.app/docs/guide_openapi/flows/

Reply:
Fixed.